### ISSUE TITLE
speexdsp compiles different files than android project used to

### ISSUE THREAD

**Issue by marijnvdwerf** [marijnvdwerf/openrct2-dependencies-android#6]
``` diff
  buffer.c
  fftwrap.c
  filterbank.c
  jitter.c
- kiss_fft.c
- kiss_fftr.c
  mdf.c
  preprocess.c
  resample.c
  scal.c
  smallft.c
```

``` diff
- -DFIXED_POINT
+ -DFLOATING_POINT
- -DUSE_KISS_FFT
  -DHAVE_STDINT_H
```

**Comment by LRFLEW** [comment]
You said you're using the ./configure script, right? The flags should be `--enable-fixed-point --with-fft=kiss`

**Comment by marijnvdwerf** [comment]
Already fixed in one of the last commits. The latter Isn't neccesary
though. But I don't know how to properly behave on each architecture.
Op zo 30 okt. 2016 om 23:55 schreef LRFLEW <notifications@github.com>

**Comment by LRFLEW** [comment]
I'd say that x86 (and it's related architecture x86_64) has good/decent floating point performance, so use floating point there. Everything else we can assume doesn't have the best floating point performance, and use fixed point there. That's how I'd do it at least.

**Comment by marijnvdwerf** [comment]
> `ANDROID_CPU_ARM64_FEATURE_FP`
Indicates that the device's CPU has a Floating Point Unit (FPU). All Android ARM64 devices must support this feature.

**Comment by marijnvdwerf** [comment]
Fixed by 0eedf24
